randovania/games/dread/json_data/Burenia.txt

```diff
@@ -2310,7 +2314,9 @@ Extra - asset_id: collision_camera_021
           Gravity Suit and After Burenia - Destroy Gravity Suit Floor
           Any of the following:
               Space Jump or Simple IBJ
-              Speed Booster and Disabled Door Lock Randomizer
+              All of the following:
+                  Speed Booster
+                  After Burenia - Gravity Suit Room Store Shinespark or Disabled Door Lock Randomizer
```

Comment:
Thinking about it, this one doesnt really need the door lock disabled on it.

Reply:
Right, because it's either storing it in the gravity suit room using the event, or if you can't access the top of the gravity suit room then you would do it from the lower part of gravity suit room, by breaking the power bomb blocks. But since we don't have an event there, we would need the "disable door lock rando" on that. And if we have that, the only thing that prevents us from storing speed in the upper part of the room again, is a missile cover.

So I guess it should be `[event] or disabled door locks rando and (missiles or lay power bomb)`